test/validators/XmlCryptoUtilities/XmlCryptoUtilities/program-repro-misc-validation-and-canon.cs

```diff
@@ -194,7 +194,7 @@ public static void SignXmlFile(string FileName, string SignedFileName, RSA Key)
         SignedXml signedXml = new SignedXml(doc);
 
         // Add the key to the SignedXml document. 
-        signedXml.SigningKey = Key;
+        signedXml.privateKey = Key;
```

Comment:
This is actually a dotnet class, and the property is correctly [SigningKey](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.xml.signedxml.signingkey?view=windowsdesktop-7.0#system-security-cryptography-xml-signedxml-signingkey)

Reply:
I knew that... I just got over-zealous in the refactor. Thanks for catching this... again, the importance of code review!